SECURITY.md

```diff
@@ -0,0 +1,19 @@
+## Coordinated Vulnerability Disclosure Policy
```

Comment:
looks ok -- better to use the updated https://github.com/crypto-org-chain/chain-main/blob/master/SECURITY.md

the hackerone inventory may also need to be later updated